hashbang/stack
# [stack-base] fix issue building gitea

Use `debian-backports` to update npm from `5.8.0+ds6-4+deb10u2` to
`6.14.8+ds-1~bpo10+1`.

Why:
To help fix an issue encountered when building the `images/gitea.tar` artifact,
which was failing to build because of an npm install error.
See #4 for more information.

What/How:
Add backported packages from the debian-backports repository to build
with a newer version of npm.

Resolved issue by following the documented debian backport instructions:
https://backports.debian.org/Instructions/

NPM Package References:
- buster npm package: https://packages.debian.org/buster/npm
- buster-backports npm package: https://packages.debian.org/buster-backports/npm

Testing:
- clean images `rm -f images/*.tar`
- build image: `make images/gitea.tar`
- verify image builds :)

Fixes #4
